index.html

```diff
@@ -9,45 +9,178 @@
+            <h2 class="salutation">Tonic</h2>
+            <ul class="list-items">
+                <li class="canopy">canopy</li> 
+                <li class="back">back end dev</li>
+                <li class="back">2015</li>
+            </ul>
+        </div>
+        <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div>
+        <ul class="buttons">
+            <li><button class="btn1" type="button">HTML</button></li>
+            <li><button class="btn1" type="button">CSS</button></li>
+            <li><button class="btn1" type="button">Javascript</button></li>
+        </ul>
+    </div>
+    <button class="resume" type="button">See Project</button>
+    </div>
+</div>
+
+<!-- The end if card 3 -->
+
+<!-- card 4 -->
+<div class="card">
+    <div class="flex-item"><img src="images/tonic.jpg" alt="works 1"></div>
+    <div class="flex-item">
+        <div class="works1">
+            <h2 class="salutation">Tonic</h2>
+            <ul class="list-items">
+                <li class="back canopy">canopy</li> 
+                <li class="back">back end dev</li>
+                <li class="back">2015</li>
+            </ul>
+        </div>
+        <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div>
+        <ul class="buttons">
+            <li><button class="btn1" type="button">HTML</button></li>
+            <li><button class="btn1" type="button">CSS</button></li>
+            <li><button class="btn1" type="button">Javascript</button></li>
+        </ul>
+    </div>
+    <button class="resume" type="button">See Project</button>
+    </div>
+</div>
+
+<!-- The end if card 4 -->
 </section>
-</footer>
+<!-- this is the end of works section -->
 
-   
+ <!-- This section gives my background -->
+<section class="about-me">
+    <p class="salutation">About Myself </p>
+    <p class="developer">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+    <p class="connect">LET'S CONNECT</p>
+
+    <section class="social">
+        <ul >
+            <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
+            <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
+            <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
+            <li><a href="https://github.com/ausiemass/portfolio"><img src="images/github.jpg" alt="GITHUB"></a></li>
+            <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
+        </ul> 
+    </section>  
+
+            
+</div>
+<button class="get-resume" type="button">Get my Resume</button>
 </div>
+<section class="skills">
```

Comment:
[OPTIONAL]
- [ ] The two `div` closing tags on lines `167 & 169` have no use in your code, kindly remove them as it hurts the professionalism of your clean code.